pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[project]
+name = "tails_server"
+version = "1.1.0"
+description = "Tails Server"
+authors = [ { name = "Government of British Columbia", email = "DItrust@gov.bc.ca" } ]
+license = { text = "Apache-2.0"}
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
```

Comment:
This shuold probably match the base image version - Python 3.12. Same for the runtime requirments just after this.